crates/sui-indexer/migrations/2022-12-01-034426_objects/up.sql

```diff
@@ -57,8 +57,9 @@ CREATE TABLE objects_history
     has_public_transfer    BOOLEAN       NOT NULL,
     storage_rebate         BIGINT        NOT NULL,
     bcs                    bcs_bytes[]   NOT NULL,
-    CONSTRAINT objects_history_pk PRIMARY KEY (checkpoint, object_id, version)
+    CONSTRAINT objects_history_pk PRIMARY KEY (object_id, version, checkpoint)
```

Comment:
this change is for `ORDER BY  object_id, version, checkpoint`?

Reply:
yes exactly and we cannot swap the order of `order by` due to different semantics here